test/e2e/dataplane/tcp_pod_connectivity.go

```diff
@@ -62,9 +71,45 @@ var _ = Describe("[dataplane] Basic TCP connectivity tests across clusters witho
 			verifyInteraction(framework.GatewayNode, framework.GatewayNode)
 		})
 	})
+
+	When("a pod can not connect to remote pod or service", func() {
```

Comment:
I added just so as to complete a basic scenario - testing connectivity and no connectivity

Reply:
But it will increase the test run time and I don't think it adds any value since we don't expect ClusterC to have submariner installed thus we don't expect connectivity for that reason so we're not really testing submariner.

Reply:
Done